e2e/pages/events/uploadDocumentsEvent.page.js

```diff
@@ -101,4 +101,42 @@ module.exports = {
   uploadCourtBundle(file) {
     I.attachFile(this.documents.courtBundle, file);
   },
+
+  uploadCaseDocuments(config, uploadSWET=true) {
+    this.selectSocialWorkChronologyToFollow(config.testFile);
+
+    if (uploadSWET) {
+      this.selectSocialWorkStatementIncludedInSWET();
+    } else {
+      this.uploadSocialWorkStatementAndGenogram(config.testFile);
+    }
+
+    this.uploadSocialWorkAssessment(config.testFile);
+    this.uploadCarePlan(config.testFile);
+
+    if (uploadSWET) {
+      this.uploadSWET(config.testFile);
+    } else {
+      this.selectSWETAsNotRequired();
+    }
+
+    this.uploadThresholdDocument(config.testFile);
+    this.uploadChecklistDocument(config.testFile);
+  },
+
+  assertCaseDocuments(hasUploadedSWET=true) {
```

Comment:
the thing is that page object is intended to represent particular page, assertions are done on different page (tab)

Reply:
True. I've changed caseDocuments fragment to a helper class as helpers offer consistent functionality as to what we want from our current caseDocuments fragment. Reverted using a page object method to enter all case documents as I think this is better suited for the helper.